admin/src/components/LinksTransactionListFormular.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="links-transaction-list-formular">
+    {{ $t('transactionlink.form_header') }}
+    <b-table striped hover :fields="fields" :items="items"></b-table>
+  </div>
+</template>
+<script>
+import { listTransactionLinksAdmin } from '../graphql/listTransactionLinksAdmin.js'
+export default {
+  name: 'LinksTransactionListFormular',
+  props: {
+    userId: { type: Number, required: true },
+  },
+  data() {
+    return {
+      fields: [
+        {
+          key: 'createdAt',
+          label: this.$t('transactionlink.created'),
+          formatter: (value, key, item) => {
+            return this.$d(new Date(value))
+          },
+        },
+        {
+          key: 'amount',
+          label: this.$t('transactionlist.amount'),
+          formatter: (value, key, item) => {
+            return `${value} GDD`
+          },
+        },
+        { key: 'memo', label: this.$t('transactionlist.memo') },
+        {
+          key: 'validUntil',
+          label: this.$t('transactionlink.expired'),
```

Comment:
This is confusing. When we have the column titled `Abgelaufen`. I suggest `Gültig bis`

Reply:
geändert